Library/ViewModels/CommentComposerViewModel.swift

```diff
@@ -42,8 +47,8 @@ public protocol CommentComposerViewModelOutputs {
   /// Emits a boolean that determines if the input area is hidden.
   var inputAreaHidden: Signal<Bool, Never> { get }
 
-  /// Emits when the input textview should resign first responder.
-  var inputTextViewResignFirstResponder: Signal<(), Never> { get }
+  /// Emits when the input textview should become first responder.
+  var inputTextViewBecomeFirstResponder: Signal<Bool, Never> { get }
```

Comment:
inputTextViewDidBecomeFirstResponder?

Reply:
I'm thinking if we say that, doesn't the same suggest a `inputTextViewDidResignFirstResponder `?

Reply:
Ah I see the `observeValues` block is either becoming or resigning first responder. Let's do:
`inputTextViewBecomeOrResignFirstResponder`?

cc: @msadoon @justinswart